mne/utils/config.py

```diff
@@ -251,12 +251,10 @@ def _open_lock(path, *args, **kwargs):
 
     lock_context = contextlib.nullcontext()  # default to no lock
 
-    if filelock is not None:
+    if filelock:
```

Comment:
Bug here -- `filelock` will be `False` if it doesn't exist, not `None`, which is causing https://github.com/pyvista/pyvista/pull/7539